crates/config/tests/missing_plugin.toml

```
@@ -0,0 +1,4 @@
+[[plugin]]
+ref = "blank_slate"
+path = "bulwark_blank_slate.wasm"
```

I don't personally do much TDD, but this is a good use-case for it. I don't know where there right place in the code to return this error is, off the top of my head, but I know what test case should trigger it.

```suggestion
ref = "does_not_exist"
path = "does_not_exist.wasm"
```

You'll also need to update this test file with a resource that points to this plugin. Something like:

```toml
[[resource]]
routes = ["/"]
plugins = ["does_not_exist"]
timeout = 25
```

The `test_load_config_missing_plugin` should attempt to load this file. It should error out, but initially won't have the correct error message. Before you write the code, you can write an assertion that says it should contain `"included plugin file not found"`. This would be expected to fail, but you can run the test with backtraces enabled and I think that should show you exactly where the existing error needs to be changed to use the new variant.